releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -2883,8 +3162,8 @@
   },
   "124685": {
     "commit": "3833d37b28095feabf3456286c4f315b062bf333",
-    "text": "kubeadm: switched kubeadm to start using the CRI client library instead of shelling out of the `crictl` binary\nfor actions against a CRI endpoint. The kubeadm deb/rpm packages will continue to install the `cri-tools`\npackage for one more release, but in you must adapt your scripts to install `crictl` manually from\nhttps://github.com/kubernetes-sigs/cri-tools/releases or a different location.\n\nThe `kubeadm` package will stop depending on the `cri-tools` package in Kubernetes 1.32, which means that\ninstalling `kubeadm` will no longer automatically ensure installation of `crictl`.",
-    "markdown": "Kubeadm: switched kubeadm to start using the CRI client library instead of shelling out of the `crictl` binary\n  for actions against a CRI endpoint. The kubeadm deb/rpm packages will continue to install the `cri-tools`\n  package for one more release, but in you must adapt your scripts to install `crictl` manually from\n  https://github.com/kubernetes-sigs/cri-tools/releases or a different location.\n  \n  The `kubeadm` package will stop depending on the `cri-tools` package in Kubernetes 1.32, which means that\n  installing `kubeadm` will no longer automatically ensure installation of `crictl`. ([#124685](https://github.com/kubernetes/kubernetes/pull/124685), [@saschagrunert](https://github.com/saschagrunert)) [SIG Cluster Lifecycle]",
+    "text": "kubeadm: Switched kubeadm to start using the CRI client library instead of shelling out of the `crictl` binary\nfor actions against a CRI endpoint. The kubeadm deb/rpm packages will continue to install the `cri-tools`\npackage for one more release, but in you must adapt your scripts to install `crictl` manually from\nhttps://github.com/kubernetes-sigs/cri-tools/releases or a different location.\n\nThe `kubeadm` package will stop depending on the `cri-tools` package in Kubernetes 1.32, which means that\ninstalling `kubeadm` will no longer automatically ensure installation of `crictl`.",
+    "markdown": "Kubeadm: Switched kubeadm to start using the CRI client library instead of shelling out of the `crictl` binary\n  for actions against a CRI endpoint. The kubeadm deb/rpm packages will continue to install the `cri-tools`\n  package for one more release, but in you must adapt your scripts to install `crictl` manually from\n  https://github.com/kubernetes-sigs/cri-tools/releases or a different location.\n  \n  The `kubeadm` package will stop depending on the `cri-tools` package in Kubernetes 1.32, which means that\n  installing `kubeadm` will no longer automatically ensure installation of `crictl`. ([#124685](https://github.com/kubernetes/kubernetes/pull/124685), [@saschagrunert](https://github.com/saschagrunert))",
```

Comment:
```suggestion
    "markdown": "Kubeadm: Switched kubeadm to start using the CRI client library instead of shelling out of the `crictl` binary\n  for actions against a CRI endpoint. The kubeadm deb/rpm packages will continue to install the `cri-tools`\n  package for one more release, but in you must adapt your scripts to install `crictl` manually from\n  https://github.com/kubernetes-sigs/cri-tools/releases or a different location.\n  \n  The `kubeadm` package will stop depending on the `cri-tools` package in Kubernetes 1.32, which means that\n  installing `kubeadm` will no longer automatically ensure installation of `crictl`. ([#124685](https://github.com/kubernetes/kubernetes/pull/124685), [@saschagrunert](https://github.com/saschagrunert)) [SIG Cluster Lifecycle]",
```